.test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored.
```

Comment:
- [ ] Kindly make sure this file is ignored by git. means it's supposed to be present here on GitHub. only locally in the repos.
Hint: You can set git to not keep track of it anymore with : `git rm test.md` and then add it to gitignore file and from there you will be good to go.